manager/config/prod/deployment_configmap.yaml

```diff
@@ -42,3 +42,4 @@ data:
   #external vault  variables
   USER_VAULT_ADDRESS: "http://vault:8200/"
   USER_VAULT_PATH: "external"
+  USER_VAULT_TOKEN: ""
```

Comment:
Any vault token should be in a secret and not in a configmap. 
I remember that there was a script that was reading a vault token from vault secret, creating a new secret to be deployed prior to the connector deployment. @rohithdv can provide more details.

Reply:
its to enable the test to pass using a dummy value

Reply:
USER_VAULT_TOKEN is relevant only for vault connector and is defined in the following way:
```
       env:
        - name: USER_VAULT_TOKEN
          valueFrom:
            secretKeyRef:
              name: user-vault-unseal-keys
              key: user-vault-root
```
What test was not passing? Perhaps the relevant secret (user-vault-unseal-keys) has not been configured.

Reply:
no. I am intentionally doing a simple dummy setup so not referencing a k8s Secret object but rather defining inline a dummy vault "" to enable the deployment test to run

Reply:
ok. I removed the field from the configmap